apps/meteor/client/views/omnichannel/agents/AgentsTable/AgentsTable.tsx

```diff
@@ -37,6 +38,9 @@ const AgentsTable = ({ reload }: { reload: MutableRefObject<() => void> }) => {
 	const query = useQuery({ text: debouncedFilter, current, itemsPerPage }, debouncedSort);
 	const { data, isSuccess, isLoading, refetch } = useAgentsQuery(query);
 
+	const [defaultQuery] = useState(hashQueryKey([query]));
+	const queryHasChanged = defaultQuery !== hashQueryKey([query]);
+
```

Comment:
`question` Why not embed it into `useAgentsQuery()`?

Reply:
To not pollute the hook 😬